movieland-parent/movieland-dao-jdbc/src/main/java/edu/eteslenko/movieland/dao/JdbcMovieDao.java

```diff
@@ -98,5 +99,9 @@ public void setMovieSelectAllQuery(String movieSelectAllQuery) {
         this.movieSelectAllQuery = movieSelectAllQuery;
     }
 
-
+    @PostConstruct
+    public void initMapping(){
+        columnMapping.put("RATING","m_rating");
+        columnMapping.put("PRICE","m_price");

```

Comment:
columnMapping = Collections.unmodifiableMap(columnMapping); 
// or Check ImmutableMap in google guava